lib/iris/tests/integration/test_intersection.py

```diff
@@ -0,0 +1,49 @@
+# (C) British Crown Copyright 2015, Met Office
+#
+# This file is part of Iris.
+#
+# Iris is free software: you can redistribute it and/or modify it under
+# the terms of the GNU Lesser General Public License as published by the
+# Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Iris is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with Iris.  If not, see <http://www.gnu.org/licenses/>.
+"""Integration tests for regridding."""
+
+from __future__ import (absolute_import, division, print_function)
+
+# Import iris.tests first so that some things can be initialised before
+# importing anything else.
+import iris.tests as tests
+import biggus
+import iris
+
+
+@tests.skip_data
+class TestLazyIntersection(tests.IrisTest):
+    def setUp(self):
+        path = tests.get_data_path(
```

Comment:
we already have a suitable file in the test data collection, can we just use this one?

I don't see what making a new file adds to this PR, does it help?